lesson_10/tests/test_log_event_unittest.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from lesson_10.src.homework_10 import log_event
+
+# Functions
+def create_expected_message_for_test(username, status):
+    return f"Login event - Username: {username}, Status: {status}"
+
+def read_login_system_file():
+    with open('login_system.log', 'r') as log_file:
+        return log_file.read()
+
+def clear_log_file():
+    open('login_system.log', 'w').close()
+
+class TestLogEventFunction(unittest.TestCase):
+
+    def test_log_event_inputs(self, name, status):
```

Comment:
```suggestion
    def log_event_inputs(self, name, status):
```
rename since it is not a test itself a can be confusions